ror-tools/src/test/scala/tech/beshu/ror/tools/RorToolsAppTest.scala

```diff
@@ -0,0 +1,172 @@
+/*
+ *    This file is part of ReadonlyREST.
+ *
+ *    ReadonlyREST is free software: you can redistribute it and/or modify
+ *    it under the terms of the GNU General Public License as published by
+ *    the Free Software Foundation, either version 3 of the License, or
+ *    (at your option) any later version.
+ *
+ *    ReadonlyREST is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *    GNU General Public License for more details.
+ *
+ *    You should have received a copy of the GNU General Public License
+ *    along with ReadonlyREST.  If not, see http://www.gnu.org/licenses/
+ */
+package tech.beshu.ror.tools
+
+import org.scalatest.BeforeAndAfter
+import org.scalatest.funsuite.AnyFunSuite
+import org.scalatest.matchers.must.Matchers.include
+import org.scalatest.matchers.should.Matchers.{equal, should}
+import tech.beshu.ror.tools.RorToolsApp.Result
+
+import java.io.{ByteArrayInputStream, ByteArrayOutputStream, IOException, PrintStream}
+import java.nio.file.Files.*
+import java.nio.file.attribute.BasicFileAttributes
+import java.nio.file.{Files, Path, Paths}
+import java.util.UUID
+
+class RorToolsAppTest extends AnyFunSuite with BeforeAndAfter {
```

Comment:
it'd be good to run the test with the different ES version (at least with the ones listed below:

<img width="820" alt="Screenshot 2025-03-01 at 19 41 52" src="https://github.com/user-attachments/assets/f034d588-56d2-46aa-a3b5-49d48bfac01a" /> 

The addition of xpack jars is not an option for sure. But I imagine we can pull a proper Elasticsearch image, copy the ES folder and run the tests. WDYT?